Source/WebCore/page/Navigator.cpp

```diff
@@ -81,7 +81,7 @@ Navigator::~Navigator() = default;
 
 String Navigator::appVersion() const
 {
-    auto* frame = this->frame();
+    SUPPRESS_UNCOUNTED_LOCAL auto* frame = this->frame();
```

Comment:
I don't think we should use SUPPRESS_UNCOUNTED_LOCAL

Reply:
Removed.